### ISSUE TITLE
mandatory-script-verify-flag-failed (Script evaluated without error but finished with a false/empty top stack element)

### ISSUE THREAD

**Issue by ValeriD** [zcash/zcash#6471]
Hello!

Describe the issue

I am following the examples and code guides. The encoded transaction is created and signed successfully: 0400008085202f8901f963206760b643786aa8099658f5ef2d95839e01dc07bde6fbcabd6bc0992240000000006b48304502210087278d93176756536bd78cfcfe6ca2e64be77fa73900a7c6edc46ea37fd36ae602203d972b659107079ee6ac6978d35eb886f6a59f3e623cec99af8d4b82f025a99a0121029173fe55c86f8e883d668c86ae22e963e05a3ba9835fdf1f24e572963ffcd018ffffff7f0280969800000000001976a914469dec9c187b6887c611e650aa92a5e01f9e8a6088acfec25b05000000001976a914baa320f7178f62db513ce171ceb17411349645c988ac00000000000000000000000000000000000000

Expected behaviour

The transaction is successfully broadcasted. Its TXID is obtained.

Actual behaviour + errors

sendrawtransaction fails with mandatory-script-verify-flag-failed (Script evaluated without error but finished with a false/empty top stack element) error message.

My node is online.
Amount for inputs was setted.
Branch id is 0xc2d6d0b4.

**Comment by daira** [comment]
Is the node synced to the current tip? Is this on mainnet or testnet? What version of zcashd?

**Comment by ValeriD** [comment]
Yes, the node is synchronised. It is on mainnet. The version of the node is **v5.3.2** and the branchId is ```c2d6d0b4```
